packages/taquito-rpc/src/taquito-rpc.ts

```diff
@@ -509,27 +508,28 @@ export class RpcClient implements RpcClientInterface {
 
   /**
    *
-   * @param options contains generic configuration for rpc calls. See examples for various available sytaxes.
+   * @param options contains generic configuration for rpc calls to specify block and version.
    *
    * @description All the information about a block
    *
    * @see https://tezos.gitlab.io/api/rpc.html#get-block-id
-   * @example getBlock() will default to /main/chains/block/head.
-   * @example getBlock({ block: head~2 }) will return an offset of 2 blocks.
-   * @example getBlock({ block: BL8fTiWcSxWCjiMVnDkbh6EuhqVPZzgWheJ2dqwrxYRm9AephXh~2 }) will return an offset of 2 blocks from given block hash..
+   * @example getBlock() will default to /main/chains/block/head?version=0. version=0 shows { kind: endorsement }
+   * @example getBlock({ version: 1 }) will return /main/chains/block/head?version=1. version=1 shows { kind: attestation }
+   * @example getBlock({ block: head~2 }) will return an offset of 2 blocks and default version 0.
    */
-  async getBlock({ block }: RPCOptions = defaultRPCOptions): Promise<BlockResponse> {
+  async getBlock({ block, version }: RPCOptions = defaultRPCOptions): Promise<BlockResponse> {
```

Comment:
There can be a design decision here: If we go with the current design, the calls to `getBlock` will need to have a version included forever. Another design might be to have an `RPC_VERSION` setting somewhere that affects this. Then the dapp can set that in one place and forget about it.